site/content/docs/03-run-time.md

```diff
@@ -162,6 +162,22 @@ Like lifecycle functions, this must be called during component initialisation.
 
 > Context is not inherently reactive. If you need reactive values in context then you can pass a store into context, which *will* be reactive.
 
+#### Reactive ContextAPI
+
+```
+<script>
+	import { setContext } from 'svelte'
+	import { writable } from 'svelte/store'
+
+	let count = writable(0)
+	setContext('count', count)
+</script>
+
+Count is: {$count}
+```
+
+More about [Reactive ContextAPI in the tutorial](https://svelte.dev/tutorial/reactive-contextapi)
+
```

Comment:
If it's going to be it's own section then it should probably be below all the context stuff with a hash link to it in the blockquote above.

Reply:
1- By "hash link" do you mean like this?

```
More about [Reactive ContextAPI in the tutorial](/tutorial/reactive-contextapi)
```

---

> If it's going to be its own section then it should probably be below all the context stuff

2- Do you mean like this? (`---`)


#### Reactive ContextAPI

---

```
<script>
	import { setContext } from 'svelte'
	import { writable } from 'svelte/store'

	let count = writable(0)
	setContext('count', count)
</script>

Count is: {$count}
```

More about [Reactive ContextAPI in the tutorial](https://svelte.dev/tutorial/reactive-contextapi)